README.md

```
@@ -5,4 +5,6 @@ Task &amp; time tracking desktop app
 
 Work In Progress
 
----
\ No newline at end of file
+---
+
+![CodeRabbit Pull Request Reviews](https://img.shields.io/coderabbit/prs/github/rrrekin/TimeCatcher?utm_source=oss&utm_medium=github&utm_campaign=rrrekin%2FTimeCatcher&labelColor=171717&color=FF570A&link=https%3A%2F%2Fcoderabbit.ai&label=CodeRabbit+Reviews)
```

markdownlint-cli2 (0.17.2)

Files should end with a single newline character

(MD047, single-trailing-newline)